components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/bean/NotificationResponseBean.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.wso2.carbon.identity.recovery.bean;
 
+import org.wso2.carbon.identity.application.common.model.ResolvedUser;
```

Comment:
The changes in this file looks not required.

Reply:
This import is required as the ResolvedUser class has been linked within the comment.